src/main/java/org/apache/sling/commons/json/JSONParserConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.sling.commons.json;
+
+import org.apache.sling.commons.json.util.ParserConfiguration;
+
+/**
+ * Configuration object for the JSON parser. The configuration is immutable.
+ */
+@Deprecated
+public class JSONParserConfiguration extends ParserConfiguration {
```

Comment:
This is a new class that was not part of the API in the previous release of the bundle. Can we also make it package-protected? Same goes for classes that were not here before.

Reply:
done for JSONML, JSONMLParserConfioguration, Property, JSONPointer, JSONPointerException, XMLParserConfiguration, XMLXsiTypeConverter, JSONParserConfiguration and stated in the readme